xplat/web/watch.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This is the new-old entry point for having a self-restarting web server running the game.